phoenix-core/src/test/java/org/apache/phoenix/query/KeyRangeMoreTest.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Collections;
 import java.util.List;
 
+import org.apache.curator.shaded.com.google.common.collect.Lists;
```

Comment:
Do we really need this import? Can we use JDK standard Lists?

Reply:
org.apache.curator.shaded.com.google.common.collect.Lists is used by old test code in this file and has nothing to do with my change. I searched in phoenix-core, there are test files using it. If we want to unify the coding style/rules and replace all the places using org.apache.curator.shaded.com.google.common.collect.Lists with JDK standard Lists, we should open an JIRA and make the change in whole code base, but it is orthogonal to and shouldn't be addressed in my change.

Reply:
I'll use "import com.google.common.collect.Lists;" instead of shaded jar